inventory/views.py

```diff
@@ -66,9 +66,31 @@ def get_context_data(self, *args, **kwargs):
         context = super(CatalogueMixin, self).get_context_data(*args, **kwargs)
         context.update({
             'item_type': self.item_type,
+            'tabs': self.get_tabs()
         })
         return context
 
+    def get_tabs(self):
+        """
+        Returns a list of dictionary objects for tab information
+        :return:
+        """
+        tabs = []
+        for item_type in Item.get_item_contenttypes():
+            if self.request.user.has_perm(f'{item_type.app_label}.view_{item_type.model}'):
+                model_class = item_type.model_class()
+                tabs.append({
+                    'verbose': model_class.__name__,
```

Comment:
Why not use the class's verbose_name instead?

Also, you can consider adding the instructions page to the tabs directly instead of having a link to it on each page.

Reply:
Don't know why I didn't consider this simple but useful thing XD